students/etwum/lesson05/mailroom_part3.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python3
+
+
+# initial list of donors
+lst_donors = [{"donor name": "Lionel Messi", "total donations": 1000000.00, "number donations": 5, "avg donation": 20000.00},
+              {"donor name": "Thierry Henry", "total donations": 500, "number donations":1, "avg donation": 500},
+              {"donor name": "Michael Jordan", "total donations": 45000, "number donations": 3, "avg donation": 15000},
+              {"donor name": "Kobe Bryant", "total donations": 8000, "number donations": 2, "avg donation": 4000}]
```

Comment:
this is not the appropriate data structure for this particular use case

for performance considerations and usability, you should be using a dictionary not a list